src/nih_wayfinding/test/karma.conf.js

```diff
@@ -85,6 +85,7 @@ module.exports = function(config) {
       'app/scripts/notifications/notifications-service.js',
       'app/scripts/routing/module.js',
       'app/scripts/routing/directions-service.js',
+      'app/scripts/routing/distance-filter.js',
```

Comment:
This is why my tests were failing, before, I think. I didn't realize it didn't use index.html for test imports.